spidermon/contrib/actions/sentry/sentry.py

```diff
@@ -0,0 +1,42 @@
+from spidermon.contrib.actions.sentry import SendMessage
+from spidermon.exceptions import NotConfigured
+
+from sentry_sdk.client import Client
+from sentry_sdk import configure_scope
+
+
+class SendSentryMessage(SendMessage):
+    def __init__(self, sentry_log_level=None, sentry_dsn=None, *args, **kwargs):
+        super(SendSentryMessage, self).__init__(*args, **kwargs)
+
+        self.sentry_log_level = sentry_log_level or self.sentry_log_level
+        self.sentry_dsn = sentry_dsn or self.sentry_dsn
+
+        if not self.fake and not self.sentry_dsn:
+            raise NotConfigured("Missing SPIDERMON_SENTRY_DSN setting")
+
+    @classmethod
+    def from_crawler_kwargs(cls, crawler):
+        kwargs = super(SendSentryMessage, cls).from_crawler_kwargs(crawler)
+        kwargs.update({
+            'sentry_dsn': crawler.settings.get('SPIDERMON_SENTRY_DSN'),
+            'sentry_log_level': crawler.settings.get('SPIDERMON_SENTRY_LOG_LEVEL'),
+        })
+        return kwargs
+
+    def send_message(self, message):
+
+        sentry_client = Client(self.sentry_dsn)
+
+        with configure_scope() as scope:
+            scope.set_extra("job_link", message.get('job_link', ''))
+            scope.set_extra("spider_name", message.get('spider_name', ''))
+            scope.set_extra("spider_stats", message.get('spider_stats', {}))
+            scope.set_extra("items_count", message.get('items_count', 0))
+
+            scope.set_extra("passed_monitors_count", message.get('passed_monitors_count', 0))
+            scope.set_extra("failed_monitors_count", message.get('failed_monitors_count', 0))
+
+            scope.set_extra("failed_monitors", message.get('failed_monitors', []))
+
+            sentry_client.capture_event({"message": message.get('title'), 'level': self.sentry_log_level}, scope=scope)
```

Comment:
This line is too long (120 chars). I believe it would be good to keep the line size close to PEP8 suggestion of 80 chars.